resources/assets/less/store/supporter-tag.less

```diff
@@ -0,0 +1,138 @@
+.store-supporter-tag {
+  &__slider {
+    .ui-slider {
+      background-color: darkgray;
+      .ui-slider-range {
+        background-color: @pink-darker;
+      }
+    }
+  }
+
+  &__user-icon {
+    margin: 8px;
+  }
+
+  &__user-search {
+    margin: 8px;
+
+    .grid-cell {
+      margin: 0 0 8px 0;
+    }
+
+    input.form-control {
+      background-color: #eee;
+      text-align: center;
+      min-width: 14em;
+    }
+  }
+
+  &--disabled {
+    .js-slider-preset {
+      background-color: #999;
+      pointer-events: none;
+      cursor: default;
+
+      &--active {
+        background-color: #999;
+      }
+    }
+
+    .ui-slider {
+      .ui-slider-handle {
+        background-color: #999;
+      }
+      &.ui-slider {
+        background-color: #999;
+        .ui-slider-range {
+          background-color: #999;
+        }
+      }
+    }
+  }
+
+  .form-control {
+    color: #555;
+    border: none;
+    box-shadow: none;
+
+    &:focus {
+      box-shadow: none;
+    }
+  }
+
+  .slider-callout {
+    position: relative;
+    background-color: #333;
+    color: #fff;
+    font-size: 12px;
+    text-align: center;
+    border-radius: 10px;
+    padding: 6px;
+    width: 220px;
+    left: 5px;
+    transform: translate3d(-50%, calc(-100% - 20px), 0);
+
+    &:after, &:before {
+      top: 100%;
+      left: 50%;
+      border: solid transparent;
+      content: " ";
+      height: 0;
+      width: 0;
+      position: absolute;
+      pointer-events: none;
+      transform: scaleX(0.5);
+    }
+
+    &:after {
+      border-color: rgba(136, 183, 213, 0);
+      border-top-color: #333;
+      border-width: 12px;
+      margin-left: -12px;
+    }
+
+    .js-discount {
+      margin-top: 2px;
+      color: #999;
+    }
+
+    .price {
+      font-size: 18px;
+      font-weight: 600;
+    }
+  }
+
+  .js-input-feedback {
+    display: inline-block; // prevent collapsing of empty block
+    font-size: 13px;
+    font-weight: 600;
+    color: #555;
+    text-align: center;
+  }
+
+  .js-slider-preset {
+    background-color: #666;
+    color: #fff;
+    cursor: pointer;
+    font-size: 12px;
+    margin: 4px;
+    height: 20px;
+    width: 32px;
+    line-height: 19px;
+    text-align: center;
+    border-radius: 10px;
+
+    &--active {
+      background-color: @pink-darker;
+    }
+  }
+
+  .price-box {
```

Comment:
better not recycle old non-bem styles because I'll be removing them. ...eventually™